spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -276,9 +278,21 @@ An <<Annotated query methods,annotated>>, <<Parameter-based automatic query meth
 A special parameter controls which query results are returned to the caller of a repository method, or in what order the results are returned:
```

Comment:
Insert language separately calling out `Restriction` as a special parameter because it can be used in a different set of places,
```suggestion
An <<Annotated query methods,annotated>> or <<Parameter-based automatic query methods,parameter-based>> query method may also have a _special parameter_ of type `Restriction`.

A special parameter controls which query results are returned to the caller of a repository method, or in what order the results are returned:
```